### ISSUE TITLE
Could not find org.jzy3d:jzy3d-jdt-core:1.0.2

### ISSUE THREAD

**Issue by douglasferlini** [FXyz/FXyz#107]
I had to configure build.grade file as below to download the dependencies:

repositories {
    mavenCentral()
    maven {
        url "https://mvnrepository.com/artifact"
    }
    maven {
        url "http://maven.jzy3d.org/releases"
    }
}

**Comment by ljramones** [comment]
I just ran into this one yesterday as I built. I used the FXyz in my code and all of a sudden I got the same issue

I've been using the fxyz3d (https://github.com/FXyz/FXyz) project in my application for a while and today as I build my project prior to a release I get this message when running the build through maven. It's built fine for 6 months until today.

Anyone know what happened to the artifacts? And does anyone know what the workaround is?

[ERROR] Failed to execute goal on project trips: Could not resolve dependencies for project com.teamgannon.trips:trips:jar:0.6.8: Failed to collect dependencies at org.fxyz3d:fxyz3d:jar:0.5.2 -> org.jzy3d:jzy3d-jdt-core:jar:1.0.2: Failed to read artifact descriptor for org.jzy3d:jzy3d-jdt-core:jar:1.0.2: Could not transfer artifact org.jzy3d:jzy3d-jdt-core:pom:1.0.2 from/to maven-default-http-blocker (http://0.0.0.0/): Blocked mirror for repositories: [jzy3d (http://maven.jzy3d.org/releases, default, releases+snapshots)] -> [Help 1]
